css/main.css

```diff
@@ -123,21 +124,21 @@ body {
 /* Small Devices, Tablets */
 @media only screen and (min-width: 600px) {
     .jumbotron-image {
-        background-position: 0px -100px;
+        background-position: 0px center;
```

Comment:
Couldn't we just change line 40 accordingly and then drop lines 126-128, 133-135 and 141? Maybe we could even drop line 63 by updating line 40?

Reply:
I agree, I've added a new commit removing all the unused css for the background jumbotron. 👍